roles/create_vms/templates/create_vm.sh.j2

```diff
@@ -1,5 +1,10 @@
 #! /bin/bash
 
+# Copy VARS.fd file related to non secure boot mode if that's the case
+{% if create_vms_disable_secure_boot %}
+cp /usr/share/edk2/ovmf/OVMF_VARS.fd /var/lib/libvirt/qemu/nvram/{{ item.name }}_no_secure_boot_VARS.fd
```

Comment:
Is there a need to copy? Can't you just point the loader at the original?

Reply:
Yes, because we need to generate one OVMF_VARS.fd file per node (copy contains `item.name`), which is later on used as argument for the `--boot` option, with `nvram=/var/lib/libvirt/qemu/nvram/{{ item.name }}_no_secure_boot_VARS.fd`. I've just done that because I saw that, by default, each VM uses a different VARS.fd file, which is created automatically.
So, I think it's fine to leave this in this way.